colorcoder/ColorService.java

```diff
@@ -0,0 +1,39 @@
+package colorcoder;
+
+public class ColorService {

```

Comment:
What services  "ColorService" provide? 

Why test code is encapsulated in ColorService?

Reply:
The primary reason for naming the class as "ColorService" was to expose a single class that contains both helper methods and test methods, after getting the comment I re-evaluated my decision and now I think I should have created a separate test class and maybe changed the ColorService methods as the following,

public static boolean testNumberToPair(int pairNumber, MajorColor expectedMajor, MinorColor expectedMinor){
	ColorPair colorPair = getColorFromNumber(pairNumber);
	System.out.println("Got pair " + colorPair.toString());
	return (colorPair.getMajor() == expectedMajor) && (colorPair.getMinor() == expectedMinor);
}